backend/clubs/views.py

```diff
@@ -3802,12 +3815,193 @@ def accept(self, request, *ages, **kwargs):
+            person=request_object.requester,
+            club=request_object.club,
+            defaults={"role": Membership.ROLE_OWNER},
+        )
+
+        if not created and membership.role != Membership.ROLE_OWNER:
+            membership.role = Membership.ROLE_OWNER
+            membership.save(update_fields=["role"])
+
+        request_object.delete()
+        return Response({"success": True})
+
+    @action(detail=False, methods=["get"], permission_classes=[IsSuperuser])
+    def old_requests(self, request, *args, **kwargs):
+        """
+        View unaddressed ownership requests that are older than a week.
+        ---
+        requestBody: {}
+        responses:
+            "200":
+                content:
+                    application/json:
+                        schema:
+                            type: array
+                            items:
+                                type: object
+                                properties:
+                                    club:
+                                        type: string
+                                    created_at:
+                                        type: string
+                                        format: date-time
+                                    email:
+                                        type: string
+                                    graduation_year:
+                                        type: integer
+                                    major:
+                                        type: array
+                                        items:
+                                            type: object
+                                            properties:
+                                                id:
+                                                    type: integer
+                                                name:
+                                                    type: string
+                                    name:
+                                        type: string
+                                    school:
+                                        type: array
+                                        items:
+                                            type: object
+                                            properties:
+                                                id:
+                                                    type: integer
+                                                name:
+                                                    type: string
+                                                is_graduate:
+                                                    type: boolean
+                                    username:
+                                        type: string
+        ---
+        """
+
+        queryset = OwnershipRequest.objects.filter(
+            withdrawn=False, created_at__lte=timezone.now() - datetime.timedelta(days=7)
+        )
```

Comment:
@rm03 Is there a reason we're limiting this to requests that are a week old? It seems reasonable that admins should be able to view all requests. (We've also had situations where someone wants to take over a club a day before the renewal deadline / Wharton applications / club fair / whatever.)

Reply:
The idea was that we give club owners a week to resolve the request before it goes to OSA; if we show all requests to OSA at once, they might resolve the request before the club owners have a chance to address it.

Reply:
Sounds good, definitely agree with the frontend part. @gabeweng do you mind tweaking this to return all requests, and then sort by `created_at` so that oldest requests show up first?